qep-322-profile-tool-point-edition.md

```diff
@@ -0,0 +1,60 @@
+# QGIS Enhancement 322: Add support for point layer edition from the profile tool
+
+**Date** 2025/02/13
+
+**Author** Jean Felder (@ptitjano), Jacky Volpes (@Djedouas)
+
+**Contact** jean dot felder at oslandia dot com, jacky dot volpes at oslandia dot com
+
+**Version** QGIS 3.44
+
+# Summary
+
+This proposal aims to enable the editing of point layers directly from the elevation profile. It will be possible to add new points, move existing ones, and delete them. Additionally, selecting points from the elevation profile will be possible. Editing directly within the elevation profile will make it easier to precisely adjust the elevation of 3D points.
+
+## Proposed Solution
+
+The main challenge of this feature is ensuring that these new functionalities do not conflict with the existing editing options in the main canvas. To make the solution as clear as possible, we propose to:
+
+- Add a new editing section to the toolbar within the elevation profile panel.
+- Display the editing icon for the layer currently selected in the elevation profile tree view.
+- Synchronize edition state between the main canvas and the elevation profile: enabling edition on a layer from the elevation profile will activate it in the main canvas, and vice versa.
+
+Tolerance will not be handled when adding or moving points. This means that the points will always be placed on the profile line.  
```

Comment:
Can you clarify what you mean here? Do you tolerance when picking the point to move? Or tolerance in allowing a user to "tear away" a point from the profile?

If the latter, wouldn't that be a handy feature? The point would keep the same x/y location, but could be translated up or down in the z axis...

Reply:
I have tried to explain it in this new version. It will be possible to ` keep the same x/y location, but could be translated up or down in the z axis`